tap_github/schemas/issue_events.json

```diff
@@ -1,1473 +1,962 @@
 {
-  "type": [
-    "null",
-    "object"
-  ],
+  "type": ["null", "object"],
   "properties": {
     "commit_id": {
-      "type": [
-        "null",
-        "string"
-      ]
+      "type": ["null", "string"]
     },
     "created_at": {
-      "type": [
-        "null",
-        "string"
-      ],
+      "type": ["null", "string"],
       "format": "date-time"
     },
     "commit_url": {
-      "type": [
-        "null",
-        "string"
-      ]
+      "type": ["null", "string"]
     },
     "url": {
-      "type": [
-        "null",
-        "string"
-      ]
+      "type": ["null", "string"]
```

Comment:
Should 'URI' format be included for URL type of fields at least for the fields, it is mentioned in the API doc?

Reply:
Some of the existing url fields were already mentioned as string without any format. So to maintain consistency, have written them as strings